src/main/scala/cromwell/webservice/ApiDataModels.scala

```diff
@@ -7,54 +7,67 @@ import cromwell.engine.backend.StdoutStderr
 
 import scala.annotation.meta.field
 
+@ApiModel(value = "WorkflowValidate")
+case class WorkflowValidateResponse
+(
+  @(ApiModelProperty@field)(required = true, value = "The validation of the workflow")
```

Comment:
"The lack of spaces is the nice touch of IntelliJ IDEA's scala plugin" this made me chuckle